README.md

```diff
@@ -1 +1,72 @@
-# My-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My-Portfolio
+
+>
+This project displays the mobile skeleton of my portfolio page. The was built with Html, Css,Vscode,Git and Github. 
+
+![screenshot](./img/mobile-skeleton.png
+)
+
+## Built With
+
+- Html
+- Css
+
+## Live Demo
+
+[Live Demo Link](https://chukwuebukavictor.github.io/My-portfolio/)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+    - VS Code
+    -Github
+
+### Setup
+   - git clone your repo
+   - cd into the folder
+   - open index.html with live server
+
+### Install
+    VS Code
+
+### Usage
+    This is a simple project to dipslay my portfolio
+
+
+## Authors
+
+👤 **Author1**
+Chukwuebuka Victor Ozoede
+
+- GitHub: [@chukwuebukaVictor](https://github.com/chukwuebukaVictor)
+- Twitter: [@OzoedeVictor](https://twitter.com/OzoedeVictor)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/chukwuebuka-ozoede-46616a219)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/chukwuebukaVictor/My-portfolio/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
### OPTIONAL
- [ ] You don't have licence in repository. You can add it. If you won't add you can delete it.